packages/cli/test/commands/__snapshots__/migrate.test.ts.snap

```diff
@@ -29,6 +21,141 @@ export function power(foo, bar) {
 }"
 `;
 
+exports[`migrate - configuing eslint > Create .eslintrc.js and install required dependencies 1`] = `
```

Comment:
We use `.input` / `.output` files concept to avoid adding files content. 

This easy to read test - you can see the list of expected files with their final content.